pkg/server/manifest_handler.go

```
@@ -37,31 +40,45 @@ func patchManifest(baseManifestData []byte, cfg *Config) []byte {
 		return baseManifestData
 	}
 
+	configRoot, err := os.OpenRoot(cfg.ConfigPath)
+	if err != nil {
+		mlog.WithError(err).Errorf("cannot open config path root %s", cfg.ConfigPath)
+		return baseManifestData
+	}
+	defer configRoot.Close()
+
 	patchedManifest := baseManifestData
```

golangci-lint (2.12.2)

[error] ineffectual assignment to patchedManifest

(ineffassign)